### PR TITLE
Add connect wallet button

### LINKED ISSUE #2297: Add 'Connect to a wallet' message or change 'Check claimable vCOW' button to 'Connect to a wallet' on the check Claim amount form
Originally reported in https://github.com/gnosis/cowswap/pull/2281#pullrequestreview-862052066

I'd like to propose to add 'Connect to a wallet' message or change 'Check claimable vCOW' button to 'Connect to a wallet' for the next screen
image

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

Fixes #2297

As defined in the issue:

Adds connect to wallet button on this screen if its not connected
Screenshot from 2022-01-27 13-01-19

Changes the text if its unsupported network
Screenshot from 2022-01-27 13-01-50

### CODE REVIEW

**Comment by biocom** [comment]
Looks good. One comment:
- I would place the 'or connect a wallet' text button below the input field.

**Comment by elena-zh** [comment]
Hey @nenadV91 , great changes! 
The same issue as was reported in closed #2309: 'Connect a wallet' button seems like in 'focused' state when navigate to this screen
image

compare: 
image


I have just realized that we have another screen where we check available claims for an account: so we show a message about claims in Ethereum, but we are currently connected to Polygon. 

image

In order to avoid this, maybe we could disable 'Check' button when connected to an ansupported network? @biocom , @alfetopito , @anxolin , @W3stside , WDYT?
image

**Comment by biocom** [comment]
> In order to avoid this, maybe we could disable 'Check' button when connected to an ansupported network?
I agree this would be good. In addition, we could reflect the same error in the primary button (Wrong network).

**Comment by anxolin** [comment]
> In order to avoid this, maybe we could disable 'Check' button when connected to an ansupported network? @biocom , @alfetopito , @anxolin , @W3stside , WDYT?

> I agree this would be good. In addition, we could reflect the same error in the primary button (Wrong network).

💯

**Review by anxolin — APPROVED** [review]
Good ideas from Elena and Michel. We can always address as new PR if we don't want to mix (handle unsupported networks)

File: src/custom/pages/Claim/ClaimAddress.tsx [hunk 1]
```diff
@@ -41,7 +43,11 @@ export default function ClaimAddress({ account, toggleWalletModal }: ClaimAddres
         <i>Note: It is possible to claim for an account, using any wallet/account.</i>
         {!account && (
           <ButtonSecondary onClick={toggleWalletModal}>
-            <Trans>or connect a wallet</Trans>
+            {error instanceof UnsupportedChainIdError ? (
+              <Trans>or connect a wallet in a supported network</Trans>
+            ) : (
+              <Trans>or connect a wallet</Trans>
```
[the remark this answers is not in the document]
Comment (nenadV91) [reply src/custom/pages/Claim/ClaimAddress.tsx]: Done

**Comment by elena-zh** [comment]
Hey @nenadV91 , changes LGTM!

Just as an enhancement, I'd navigate a user to this screen 
image
from every claim screen when change a network to an unsupported one. 
Currently, it navigates to this screen
image

But anyways, let me know please if I need to create a separate issue for this. 

Thanks

**Comment by nenadV91** [comment]
@elena-zh Updated

**Review by elena-zh — APPROVED** [review]
LGTM now!

**Pull request merged**